runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordState.java

```diff
@@ -109,11 +108,14 @@ void refresh()
 			.partyMax(lastPresence.getPartyMax())
 			.partySize(party.getMembers().size());
 
-		if (!party.isInParty() || party.isPartyOwner())
+		if (party.isInParty())
 		{
-			// This is only used to identify the invites on Discord's side. Our party ids are the secret.
-			presenceBuilder.partyId(partyId.toString());
-			presenceBuilder.joinSecret(party.getLocalPartyId().toString());
+			// Create a partyID for Discord to consider us part of this party and thus display our avatars on Discord side
+			presenceBuilder.partyId(DiscordPlugin.toDiscordPartyId(party.getPartyId()).toString());
+
+			// It was decided to allow any party members to invite / be requested to join a party
+			//if (party.isPartyOwner())
```

Comment:
As adam said in other comment, why are you leaving commented out code aroud. Also move this and the other identical party check in this class to common method